sw/vendor/riscv-opcodes/opcodes-rep

```diff
@@ -1,2 +1,3 @@
-frep            rd rs1 imm12 rm 6..2=0x02 1..0=3
+frep.o          rs1 imm12 stagger_max stagger_mask 7=1 6..2=0x02 1..0=3
+frep.i          rs1 imm12 stagger_max stagger_mask 7=0 6..2=0x02 1..0=3
```

Comment:
Just a sanity question: Shouldn't the hardware-generated opcodes also change if they were in sync?